packages/material/src/controls/MaterialOneOfEnumControl.tsx

```diff
@@ -31,13 +31,13 @@ import {
   rankWith,
 } from '@jsonforms/core';
 import { withJsonFormsOneOfEnumProps } from '@jsonforms/react';
-import { MuiSelect } from '../mui-controls/MuiSelect';
+import { MuiAutocomplete } from '../mui-controls/MuiAutocomplete';
 import { MaterialInputControl } from './MaterialInputControl';
 
 export const MaterialOneOfEnumControl = (props: ControlProps & OwnPropsOfEnum) => (
   <MaterialInputControl
     {...props}
-    input={MuiSelect}
+    input={MuiAutocomplete}
```

Comment:
I think I would prefer making `MuiAutocomplete` the default here and in the normal enum but making it configurable via `options: { autoComplete: false }`. Don't forget to check the `config` too.